schedule.yaml

```diff
@@ -85,9 +87,158 @@ league:
+# Top 2 continue in the same bracket; bottom 2 go down a bracket.
+# If a team finishes in the bottom 2 in a match in the Lower bracket, they are out.
+# All teams start in the Upper bracket.
+
+static_knockout:
+  teams_per_arena: 4
+  
+  matches:
+    0:  # Upper Round 1
+      0:
+        arena: main
+        display_name: Upper Round 1 Match 1
+        start_time: 2024-04-13 12:44:00+01:00
+        teams: [S13, S21, S20, null]
+      1:
+        arena: main
+        display_name: Upper Round 1 Match 2
+        start_time: 2024-04-13 12:49:00+01:00
+        teams: [S14, S22, S19, null]
+      2:
+        arena: main
+        display_name: Upper Round 1 Match 3
+        start_time: 2024-04-13 12:54:00+01:00
+        teams: [S15, S23, S18, S26]
+      3:
+        arena: main
+        display_name: Upper Round 1 Match 4
+        start_time: 2024-04-13 12:59:00+01:00
+        teams: [S16, S24, S17, S25]
+    1:  # Upper Round 2
+      0:
+        arena: main
+        display_name: Upper Round 2 Match 1
+        start_time: 2024-04-13 13:14:00+01:00
+        teams: ['000', '011', '030', '021']
+      1:
+        arena: main
+        display_name: Upper Round 2 Match 2
+        start_time: 2024-04-13 13:19:00+01:00
+        teams: ['001', '010', '031', '020']
+    2:  # Lower Round 1
+      0:
+        arena: main
+        display_name: Lower Round 1 Match 1
+        start_time: 2024-04-13 13:24:00+01:00
+        teams: ['003', '032', '013', '022']
+      1:
+        arena: main
+        display_name: Lower Round 1 Match 2
+        start_time: 2024-04-13 13:29:00+01:00
+        teams: ['002', '033', '012', '023']
+    3:  # Upper Round 3
+      0:
+        arena: main
+        display_name: Upper Round 3 Match 1
+        start_time: 2024-04-13 13:34:00+01:00
+        teams: [S1, '111', S8, S9]
+      1:
+        arena: main
+        display_name: Upper Round 3 Match 2
+        start_time: 2024-04-13 13:39:00+01:00
+        teams: [S2, '101', S7, S10]
+      2:
+        arena: main
+        display_name: Upper Round 3 Match 3
+        start_time: 2024-04-13 13:44:00+01:00
+        teams: [S3, '110', S6, S11]
+      3:
+        arena: main
+        display_name: Upper Round 3 Match 4
+        start_time: 2024-04-13 13:49:00+01:00
+        teams: [S4, '100', S5, S12]
+    4:  # Lower Round 2
+      0:
+        arena: main
+        display_name: Lower Round 2 Match 1
+        start_time: 2024-04-13 13:54:00+01:00
+        teams: ['200', '313', '103', '302']
+      1:
+        arena: main
+        display_name: Lower Round 2 Match 2
+        start_time: 2024-04-13 13:59:00+01:00
+        teams: ['201', '312', '102', '303']
+      2:
+        arena: main
+        display_name: Lower Round 2 Match 3
+        start_time: 2024-04-13 14:04:00+01:00
+        teams: ['210', '333', '113', '322']
+      3:
+        arena: main
+        display_name: Lower Round 2 Match 4
+        start_time: 2024-04-13 14:09:00+01:00
+        teams: ['211', '332', '112', '323']
```

Comment:
These teams don't line up with what's in the sheet, however the sheet appears wrong since it suggests that `332` would appear twice here. What should the fourth team here be?

Reply:
As you spotted in the sheet, it should have been P4 in #111 on the sheet, which means I think was actually correct, even though it didn't match the sheet? I've changed the sheet now if you want to rerun your check script.

Reply:
Yup, all good now.